website/integrations/services/slack/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Slack
+---
+
+<span class="badge badge--secondary">Support level: authentik</span>
+
+## What is Slack
+
+> Slack is a platform for collaboration, with chat and real-time video capabilities.
+>
+> -- https://slack.com
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `slack.company` is the FQDN of the Slack install.
+-   `authentik.company` is the FQDN of the authentik install.
+
+## authentik configuration
+
+### Step 1. Create custom property mappings
+
+Your Slack integration requires property mappings for `User.Email` and `User.Username` so that authetnik can retrieve and map these values from Slack.
```

Comment:
We have one additional one for our slack workspace. The SAML attribute name is `first_name` and the expression is:
```python
parts = request.user.name.split(" ")
if len(parts) < 1:
  return request.user.name
return parts[0]
```

Not sure if it's required though.

Reply:
@BeryJu can you please confirm (or deny lol) if this property mapping is required? Thanks!

Reply:
It's not required (see https://slack.com/help/articles/205168057-Custom-SAML-single-sign-on, and we should probably link that page in this doc too)